### PR TITLE
Fix dmn-drools-quarkus-metrics pom and readme

### PR DESCRIPTION
**Pull request by r00ta**
Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

**WARNING! Please make sure you are opening your PR against `main` branch!**

- [ ] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [ ] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting main: `[0.9.x] KOGITO-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Pull Request</b>  
  Please add comment: <b>Jenkins retest this</b>
 
* <b>Quarkus LTS checks</b>  
  Please add comment: <b>Jenkins run LTS</b>

* <b>Native checks</b>  
  Please add comment: <b>Jenkins run native</b>

</details>

### CODE REVIEW

File: dmn-drools-quarkus-metrics/pom.xml [hunk 1]
```diff
@@ -109,7 +109,7 @@
         <executions>
           <execution>
             <id>copy-resources</id>
-            <phase>prepare-package</phase>
+            <phase>install</phase>
```
Comment (gitgabrio) [inline dmn-drools-quarkus-metrics/pom.xml]: Thanks @r00ta !
One question: is not this modification needed also for  "copy-docker-compose-for-tests" execution ?

**Comment by gitgabrio** [comment]
@r00ta 
Is this not needed also for  dmn-drools-springboot-metrics ?

**Comment by gitgabrio** [comment]
@r00ta 
While you're at that, could you also fix maven clean so that also files inside docker-compose get deleted ? Currently, mvn clean remove `target` directory but leaves `docker-compose` as it is from previous build

**Comment by r00ta** [comment]
> @r00ta While you're at that, could you also fix maven clean so that also files inside docker-compose get deleted ? Currently, mvn clean remove `target` directory but leaves `docker-compose` as it is from previous build

done

File: dmn-drools-quarkus-metrics/pom.xml (hunk 1 above)
Comment (r00ta) [reply dmn-drools-quarkus-metrics/pom.xml]: I was actually wrong. 
```
            <phase>prepare-package</phase>
```
is running fine and we should keep it. 

Restored

**Comment by r00ta** [comment]
> @r00ta Is this not needed also for dmn-drools-springboot-metrics ?

Yes good point. Done

**Review by danielezonca — APPROVED** [review]

**Review by gitgabrio — APPROVED** [review]
HI @r00ta 
thanks for the PR: LGTM

File: dmn-drools-quarkus-metrics/pom.xml (hunk 1 above)
Comment (gitgabrio) [reply dmn-drools-quarkus-metrics/pom.xml]: Hi @r00ta 
On second thought, t is not running fine.

This is order of execution of `mvn clean install -DskipTests` 

`[INFO] --- maven-clean-plugin:3.1.0:clean (default-clean) @ dmn-drools-quarkus-metrics ---`
`[INFO] --- maven-enforcer-plugin:3.0.0-M2:enforce (enforce-versions) @ dmn-drools-quarkus-metrics ---`
`[INFO] --- maven-enforcer-plugin:3.0.0-M2:enforce (enforce-java-version) @ dmn-drools-quarkus-metrics ---`
`[INFO] --- maven-enforcer-plugin:3.0.0-M2:enforce (enforce-maven-version) @ dmn-drools-quarkus-metrics ---` 
`[INFO] --- maven-checkstyle-plugin:3.0.0:check (default) @ dmn-drools-quarkus-metrics ---`
`[INFO] --- buildnumber-maven-plugin:1.4:create (get-scm-revision) @ dmn-drools-quarkus-metrics ---`
`[INFO] --- formatter-maven-plugin:2.13.0:format (default) @ dmn-drools-quarkus-metrics ---`
`[INFO] --- impsort-maven-plugin:1.5.0:sort (default) @ dmn-drools-quarkus-metrics ---`
`[INFO] --- maven-resources-plugin:2.5:resources (default-resources) @ dmn-drools-quarkus-metrics ---`
`[INFO] --- maven-compiler-plugin:3.8.1:compile (default-compile) @ dmn-drools-quarkus-metrics ---`
`[INFO] --- maven-resources-plugin:2.5:testResources (default-testResources) @ dmn-drools-quarkus-metrics ---`
`[INFO] --- maven-compiler-plugin:3.8.1:testCompile (default-testCompile) @ dmn-drools-quarkus-metrics ---`
`[INFO] --- maven-surefire-plugin:3.0.0-M5:test (default-test) @ dmn-drools-quarkus-metrics ---`
`[INFO] --- maven-resources-plugin:2.5:copy-resources (copy-resources) @ dmn-drools-quarkus-metrics ---`
`[INFO] skip non existing resourceDirectory /home/gcardosi/Developing/git/kogito-examples/dmn-drools-quarkus-metrics/target/classes/META-INF/resources/monitoring/dashboards`
`[INFO] .....`

The invocation of `maven-resources-plugin` does not find anything, so nothing get copied to `grafana/provisioning/dashboards`.

If, on the other side, you run  `mvn clean install`, the `maven-resources-plugin` get invoked twice, the second time by test execution, and this time it finds resources to be copied.
Of course, resources must be generated and available even skipping the test, that's why I think the configuration should be changed (and probably `install` phase is right)

**Pull request merged**